### PR TITLE
successful connect to devnet test

### PR DESCRIPTION
**Pull request by svv232**
this is a child pr for #15928

### CODE REVIEW

**Comment by svv232** [comment]
!ci-build-me

**Comment by svv232** [comment]
!ci-build-me

**Comment by svv232** [comment]
!ci-build-me

**Comment by svv232** [comment]
!ci-build-me

File: automation/scripts/gen-keys-ledger.sh [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #! /bin/bash
-MINA_DAEMON_IMAGE="minaprotocol/mina-daemon:1.3.2beta2-release-2.0.0-6f9d956-focal-berkeley"
+MINA_DAEMON_IMAGE="minaprotocol/mina-daemon:1.3.2beta2-release-2.0.0-6f9d956-focal-devnet"
```
Comment (dkijania) [inline automation/scripts/gen-keys-ledger.sh:2]: There is no such image as ...-devnet . This is an old image, which leads to a conclusion that it is a old script (not anymore valid). I would revert this change

**Review by dkijania — APPROVED** [review]

File: automation/scripts/gen-keys-ledger.sh (hunk 1 above)
Comment (svv232) [reply automation/scripts/gen-keys-ledger.sh:2]: i see, reverting then

**Comment by svv232** [comment]
!ci-build-me

**Comment by svv232** [comment]
!ci-build-me
